ddr/ddr/GameScene.swift

```diff
@@ -63,7 +106,7 @@ class GameScene: SKScene, SKPhysicsContactDelegate {
   }
   
   func beginSpawningBats() {
-      let wait = SKAction.wait(forDuration: 5, withRange: 2)
+      let wait = SKAction.wait(forDuration: 2, withRange: 2)
```

Comment:
Might we want to pull these out into class constants?

Reply:
These will get pulled out when we restructure for gameplay progression. Fine for now. Resolved